### ISSUE TITLE
[BUG] Accessibility issues

### ISSUE THREAD

**Issue by elichad** [RSEToolkit/rse-competencies-toolkit-webapp#719]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Describe the bug

Reported by @mzhc13 :
**See sub issues**

### Expected Behavior

WAVE report passes

### Environment

```markdown
- OS:
- Browser:
- Python version:
- Software versions:
```

**Comment by elichad** [comment]
Tried to edit the CSS with no luck - don't know how the system works.

But our suggestions are:
`--ar-body-color` should be updated to `--ar-gray-800` 
`--ar-secondary-color`  and `ar-breadcrumb-color` should be updated to `ar-gray-700`
The "Hello, <name>" at the top of the page should use `text-muted` class instead of an opacity class
